cli/src/cpp/usages.hpp

```diff
@@ -32,6 +32,23 @@ static const char COMPARE_USAGE[] =
       fastddsqosprof <file_1> compare <file_2>
 )";
 
+static const char DATAREADER_DEFAULT_PROFILE_USAGE[] =
```

Comment:
Maybe this usage can be unified similarly to the `REALIBILITY_KIND_QOS_USAGE`, generalizing referring to endpoint. Though it can be merged as it is.

Reply:
Yes, I like your suggestion. `usages.hpp` length will increase significantly, so the less redundant the better